flake.nix

```diff
@@ -143,11 +143,9 @@
           };
 
           settings = {
```

Comment:
Maybe it's worth leaving a note that "since this will go into a user's config, and they may not necessarily be a trusted-user, we should avoid configuring settings that only trusted-users are allowed to configure"

Reply:
```suggestion
          settings = {
            # Note: homeManager configures the user's environment, so only set restricted Nix options.
```

Reply:
How's that, @cole-h?